src/components/Cascader.tsx

```diff
@@ -92,6 +101,7 @@ const CascaderImpl: ComponentImpl<Static<typeof CascaderPropsSchema>> = (
 
   const onChange = (value: (string | string[])[]) => {
     _setValue(value);
+    callbackMap?.onChange();
```

Comment:
Please change to `callbackMap?.onChange?.()`. Otherwise, if the user selects other events and `onChange` would be `undefined`.